hedera-node/src/main/java/com/hedera/services/config/FileNumbers.java

```diff
@@ -37,6 +38,7 @@
 	private long applicationProperties = UNKNOWN_NUMBER;
 
 	private long systemFileCutoff = UNKNOWN_NUMBER;
+	private long updateFeatureFile = ApplicationConstants.UPDATE_FEATURE_FILE_ACCOUNT_NUM;
```

Comment:
`ApplicationConstants` is deprecated; **all** properties should now get default values from _src/main/resources/bootstrap.properties_. Addressed in #658.